docs/falcon_rocket.md

```diff
@@ -0,0 +1,11 @@
+## SpaceX-like Falcon Rocket
+
+I've assembled a Falcon rocket that can learn to land itself on the autonomous floating drone ship. I reused a part of the LunarLander (literally, the lander) from <a href="https://github.com/openai/gym">OpenAI Gym</a> and added some more elements so that it makes the task more difficult and more interesting.
```

Comment:
Maybe don't use the first person here, no "I", just describe it and maybe add a reference to your Github or website at the bottom of the page!

Suggested:

This Falcon rocket lander environment encourages landing on an autonomous floating drone ship. It uses part of the LunarLander environment as a base (from <a href="https://github.com/openai/gym">OpenAI Gym</a> ) and adds some elements that make the task more difficult and interesting as follows.